package.json

```diff
@@ -16,6 +16,7 @@
     "base-64": "^1.0.0",
     "cross-fetch": "^4.0.0",
     "lodash": "^4.17.21",
+    "react": "^18.2.0",
```

Comment:
is this needed?

Reply:
This part of the imports caused a failure in the test without it.
![image](https://github.com/amplication/amplication-testing/assets/39680385/b831b565-b297-4019-a318-95e871b34d5c)